package.json

```diff
@@ -77,6 +77,7 @@
     "expect": "^1.20.2",
     "is-promise": "^2.1.0",
     "jest": "^20.0.1",
+    "jest-cli": "^20.0.4",
```

Comment:
Do we really need this? because `jest-cli` already exposed by the main `jest` package

https://github.com/facebook/jest/blob/master/packages/jest/package.json
```js
  "dependencies": {
    "jest-cli": "^20.0.4"
  },
  "bin": {
    "jest": "./bin/jest.js"
  }
```

https://github.com/facebook/jest/blob/master/packages/jest/bin/jest.js
```js
require('jest-cli/bin/jest');
```

Reply:
It stopped working for me without it...

Reply:
I tested this locally. Worked even after I removed `jest-cli`. Maybe try a fresh install?

Reply:
Yeah, after doing `rm -rf node_modules` and reinstalling everything it work. I'm wondering if npm v10 will fix that annoying behavior or if this will remain the trademark for it 🙄

Reply:
Believe me it is worse on windows. I actually gave up and switched to yarn. I dunno if the latest release fixed some of the issues.

Reply:
yeah still happens sometimes 😢

Reply:
Every time I install/uninstall dependency I'm getting

```
module.js:487
    throw err;
    ^

Error: Cannot find module '/Users/okonet/Projects/OSS/lint-staged/node_modules/jest-cli'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at getJest (/Users/okonet/Projects/OSS/lint-staged/node_modules/jest/node_modules/jest-cli/build/cli/getJest.js:22:12)
    at Object.run (/Users/okonet/Projects/OSS/lint-staged/node_modules/jest/node_modules/jest-cli/build/cli/index.js:39:48)
    at Object.<anonymous> (/Users/okonet/Projects/OSS/lint-staged/node_modules/jest/node_modules/jest-cli/bin/jest.js:14:25)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
```

so I think I'd better stick with `jest-cli` dependency for now.

Reply:
hmmm weird 🤔